go.mod

```diff
@@ -3,27 +3,45 @@ module github.com/thrasher-corp/gocryptotrader
 go 1.12
 
 require (
+	cloud.google.com/go v0.51.0 // indirect
+	cloud.google.com/go/storage v1.5.0 // indirect
+	dmitri.shuralyov.com/gpu/mtl v0.0.0-20191203043605-d42048ed14fd // indirect
```

Comment:
It looks like doing a go get golangci-lint has updated the go.mod/go.sum files, we make this optional and not a requirement as some people would like to compile the binary without the golangci files. The grpc/protobuffs newer versions are fine however

Reply:
Sorry, I am not sure what you mean. I am a newbie in Go, just starting up. Do I need to change anything?

Reply:
go modules (and the go.mod file) are go handles dependency management 
these additions will cause go get to pull down these package 

a common thing people new to go do is run "go get" on optional tools they want to install while inside a go module enabled project

such as 
```
go get -u github.com/golangci/golangci-lint
```
running this while inside the gct directory will cause it to add golangci-lint and its dependencies to gct's list

easiest thing here would most likely be to revert them both back to master

```
git checkout master -- go.mod go.sum
```

Reply:
gotcha, thanks
I'll fix it